server/server/views/admin.py

```diff
@@ -0,0 +1,52 @@
+from cornice.resource import resource, view
+from pyramid.security import Allow
+
+from server.models import model_to_dict
+from server.models.user import User
+from server.models.user_status import UserStatus
+from server.models.role import Role
+
+
+@resource(collection_path='/admin-page/users', path='/admin-page/users/{id}',
+          renderer='json', cors_origins=('http://localhost:3000',))
+class AdminView(object):
+
+    def __init__(self, request, context=None):
+        self.request = request
+        self.context = context
+
+    def __acl__(self):
+        return [(Allow, 'role:admin', 'admin')]
+
+    @view(permission="admin")
+    def collection_get(self):
+        users_list = User.get_all(self.request)
+        roles_list = Role.get_all(self.request)
+        statuses_list = UserStatus.get_all(self.request)
+        users_dict = []
+        roles_dict = []
+        statuses_dict = []
+        for obj in users_list:
+            temp_dict = model_to_dict(obj)
+            temp_dict['status_str'] = obj.user_statuses.status
+            users_dict.append(temp_dict)
+        roles_dict = [model_to_dict(obj) for obj in roles_list]
+        statuses_dict = [model_to_dict(obj) for obj in statuses_list]
+        response = {}
```

Comment:
You can add arrays to response when you define it. Also, you can immediately return it, without a response variable.